tests/fixtures.py

```diff
@@ -64,6 +68,13 @@ def _prepare_root_module(path):
       yield tmp_path
   else:
     # if TFTEST_COPY is not set, just return the same path
+
+    if unwanted_files := ignore_patterns(path, os.listdir(path=path)):
```

Comment:
Actually, what is done here, is when we do NOT copy the files into temporary directory, just check if any of them are present, as this may affect the test results

Reply:
what I wrote above :) if any are in the unwanted list and not copied, raise an error